database/postgresconn/postgresconn_test.go

```diff
@@ -319,74 +282,6 @@ func TestWithSchema(t *testing.T) {
 	})
 }
 
-func TestMigrationTableOption(t *testing.T) {
```

Comment:
No longer a valid use case, we removed the ability to provide migration table name and schema as part of the url